<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageMyMobile_DesktopClientClass</class>
 <widget class="QMainWindow" name="ManageMyMobile_DesktopClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>520</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Mobii</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	font-family: &quot;Segoe UI&quot;;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>750</width>
     <height>520</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="TopPanelFrame">
      <property name="styleSheet">
       <string notr="true">#TopPanelFrame {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #f6f7fa, stop: 1 #dadbde);

}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="TopMenu" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="TopInfosBar" native="true">
            <property name="styleSheet">
             <string notr="true">#TopInfosBar {
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgb(255, 55, 55), stop: 1 #e0eff9);
}</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="leftMargin">
              <number>16</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QLabel" name="logoImage">
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>140</width>
                 <height>32</height>
                </size>
               </property>
               <property name="pixmap">
                <pixmap resource="mobii.qrc">:/common/Resources/logo_verbose_flat_small.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="MainInfoLabel">
               <property name="text">
                <string>Vous n'êtes pas connecté au serveur Mobii.</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="DisconnectLink">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <weight>75</weight>
                 <bold>true</bold>
                 <underline>true</underline>
                </font>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {border: none;color: rgb(0, 85, 255);text-decoration: underline;font-weight: bold;}QPushButton:hover {color: rgb(0, 170, 255)}QPushButton[enabled=&quot;false&quot;] {rgb(140, 140, 140)}</string>
               </property>
               <property name="text">
                <string>Déconnecter</string>
               </property>
               <property name="icon">
                <iconset resource="mobii.qrc">
                 <normaloff>:/buttons/Resources/power.png</normaloff>:/buttons/Resources/power.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="TopMenuActionsPanel" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: rgba(255, 255, 255, 0);
	border: none;
	padding: 0px 10px;
	height: 100%;
	font: bold 13px &quot;Segoe UI&quot;;
}

QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(59, 122, 255, 255), stop:0.106599 rgba(59, 122, 255, 255), stop:0.108333 rgba(255, 255, 255, 0), stop:1 rgba(255, 255, 255, 0))
}

QPushButton[checked=&quot;true&quot;] {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(59, 122, 255, 255), stop:0.106599 rgba(59, 122, 255, 255), stop:0.107402 rgba(160, 209, 255, 255), stop:1 rgba(193, 239, 239, 255));
}</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="TopHomeButton">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Page d'accueil de Mobii</string>
               </property>
               <property name="text">
                <string>Accueil</string>
               </property>
               <property name="icon">
                <iconset resource="mobii.qrc">
                 <normaloff>:/panels/Resources/home.png</normaloff>:/panels/Resources/home.png</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="TopNewConnectionButton">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Se connecter à un smartphone</string>
               </property>
               <property name="text">
                <string>Nouvelle
connexion</string>
               </property>
               <property name="icon">
                <iconset resource="mobii.qrc">
                 <normaloff>:/panels/Resources/phone.png</normaloff>:/panels/Resources/phone.png</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="TopConnectionEstablishedMenu" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="TopMobileInfosPanelButton">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Informations pratiques sur votre smartphone</string>
                  </property>
                  <property name="text">
                   <string>Informations
mobile</string>
                  </property>
                  <property name="icon">
                   <iconset resource="mobii.qrc">
                    <normaloff>:/panels/Resources/phone.png</normaloff>:/panels/Resources/phone.png</iconset>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="autoExclusive">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="TopSmsPanelButton">
                  <property name="toolTip">
                   <string>Messages SMS</string>
                  </property>
                  <property name="text">
                   <string>SMS</string>
                  </property>
                  <property name="icon">
                   <iconset resource="mobii.qrc">
                    <normaloff>:/panels/Resources/chat.png</normaloff>:/panels/Resources/chat.png</iconset>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="autoExclusive">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="TopContactPanelButton">
                  <property name="toolTip">
                   <string>Contacts</string>
                  </property>
                  <property name="text">
                   <string>Contacts</string>
                  </property>
                  <property name="icon">
                   <iconset resource="mobii.qrc">
                    <normaloff>:/panels/Resources/contacts.png</normaloff>:/panels/Resources/contacts.png</iconset>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="autoExclusive">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="TopCalendarPanelButton">
                  <property name="toolTip">
                   <string>Calendrier</string>
                  </property>
                  <property name="text">
                   <string>Agenda</string>
                  </property>
                  <property name="icon">
                   <iconset resource="mobii.qrc">
                    <normaloff>:/panels/Resources/calendar.png</normaloff>:/panels/Resources/calendar.png</iconset>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="autoExclusive">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="TopImagePanelButton">
                  <property name="toolTip">
                   <string>Photos et images</string>
                  </property>
                  <property name="text">
                   <string>Fichiers</string>
                  </property>
                  <property name="icon">
                   <iconset resource="mobii.qrc">
                    <normaloff>:/panels/Resources/file.png</normaloff>:/panels/Resources/file.png</iconset>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="autoExclusive">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="TopSettingsButton">
               <property name="toolTip">
                <string>Modifier les options de Mobii</string>
               </property>
               <property name="icon">
                <iconset resource="mobii.qrc">
                 <normaloff>:/panels/Resources/tools.png</normaloff>:/panels/Resources/tools.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="TopLegalNoticeButton">
               <property name="toolTip">
                <string>Mentions légales</string>
               </property>
               <property name="text">
                <string>?</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QStackedWidget" name="ContentWidgetStack">
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mobii.qrc"/>
 </resources>
 <connections/>
</ui>
